### PR TITLE
Adding getOption method to the cli dispatcher. Adding options into the task handler.

### PR DESCRIPTION
**Pull request by gguridi**
It's weird that on command line tasks the dispatcher has the options but doesn't pass them onto the task action so they can be accessed directly.

I've also added the getOption method to the command line dispatcher so you can retrieve one specific option, passing a filter to ensure nothing dangerous comes by.

---

This change is [Reviewable](https://reviewable.io/reviews/phalcon/cphalcon/12219)

### CODE REVIEW

File: phalcon/cli/dispatcher.zep [hunk 1]
```diff
@@ -151,8 +151,37 @@ class Dispatcher extends \Phalcon\Dispatcher implements DispatcherInterface
...
+			this->{"_throwDispatchException"}("A dependency injection object is required to access the 'filter' service", \Phalcon\Dispatcher::EXCEPTION_NO_DI);
+		}
+		let filter = <FilterInterface> dependencyInjector->getShared("filter");
+		return filter->sanitize(optionValue, filters);
+	}
+
 	public function callActionMethod(handler, string actionMethod, array! params = [])
 	{
-		return call_user_func_array([handler, actionMethod], [params]);
+		return call_user_func_array([handler, actionMethod], [params, this->_options]);
```
Comment (sergeyklay) [inline phalcon/cli/dispatcher.zep]:
```zephir
var options;

let options = this->_options;
return call_user_func_array([handler, actionMethod], [params, options]);
```

File: phalcon/cli/dispatcher.zep [hunk 2]
```diff
@@ -151,8 +151,37 @@ class Dispatcher extends \Phalcon\Dispatcher implements DispatcherInterface
...
 
+	/**
+	 * Gets an option by its name or numeric index
+	 *
+	 * @param  mixed param
+	 * @param  string|array filters
+	 * @param  mixed defaultValue
+	 * @return mixed
+	 */
+	public function getOption(option, filters = null, defaultValue = null)
```
Comment (sergeyklay) [inline phalcon/cli/dispatcher.zep]:
```zephir
public function getOption(option, filters = null, defaultValue = null) -> var
```

and remove `@return mixed`

**Comment by gguridi** [comment]
Added the changes suggested to match the code standards.

**Comment by Jurigag** [comment]
Rebase and squash maybe ?

**Comment by gguridi** [comment]
Squashed. Thanks for reminding me that! :smile:

**Comment by sergeyklay** [comment]
@gguridi Well, as you can see tests failed. Also could you please update `CHANGELOG.md`.

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**